lib/smart_answer_flows/overseas-passports/outcomes/getting_your_passport/_collect_in_person.govspeak.erb

```diff
@@ -0,0 +1,2 @@
+
+You must collect in person.  If you’re unable to, someone else can go on your behalf.
```

Comment:
OK. I've always felt free to question the wording of content, but I'll leave it up to you.